wrappers/python/nntile/model/gpt2_block.py

```diff
@@ -0,0 +1,146 @@
+# @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+#                              (Skoltech), Russia. All rights reserved.
+#                2023-present Artificial Intelligence Research Institute
+#                              (AIRI), Russia. All rights reserved.
+#
+# NNTile is software framework for fast training of big neural networks on
+# distributed-memory heterogeneous systems based on StarPU runtime system.
+#
+# @file wrappers/python/nntile/model/gpt2_block.py
+# GPT2Block submodule of NNTile Python package
+#
+# @version 1.0.0
```

Comment:
version 1.1.0

Reply:
What about `@version 1.0.0` in all source files of release tag `v1.1.0`?

Reply:
That's insane. My browser crashed, lol.

![image](https://github.com/user-attachments/assets/0754a61a-485a-4948-a971-b347de8e03d1)